gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/NackAckPreparationService.java

```diff
@@ -22,7 +22,7 @@
 @Slf4j
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class NackAckPreparationService {
+public class NackAckPreparationService implements NackAckPrepInterface {
```

Comment:
Is an interface required here? Is there likely to be > 1 concrete implementations for `NackAckPrepInterface`? Just a question, not too sure what the thinking is around this! 😄